sympy/assumptions/assume.py

```diff
@@ -335,6 +339,8 @@ def _(func):
         return _
 
     def __call__(self, *args):
+        if self._arity is not None and len(args) != self._arity:
+                raise TypeError(f"{type(self)} requires {self._arity} arguments but only {len(args)} were given")
```

Comment:
You can probably remove the word "only" from the error message. 

"Q.positive requires 1 argument but only 2 were given" sounds strange.

Also, I think `type(self)` might return `sympy.assumptions.predicates.order.PositivePredicate` which would make the error message longer than it needs to be. Maybe try this:
```
raise TypeError(f"{self} requires {self._arity} arguments but {len(args)} were given")
```